GitExtensionsVSIX/source.extension.vsixmanifest

```diff
@@ -13,7 +13,9 @@
     </Metadata>
 
     <Installation>
-        <InstallationTarget Version="[14.0,15.0)" Id="Microsoft.VisualStudio.Community" />
+        <InstallationTarget Version="[14.0,16.0)" Id="Microsoft.VisualStudio.Community" />
```

Comment:
I vaguely remember having some issues with that, but maybe it was part of the pre-release. However I can no longer test this because Git extensions doesn't compile with VS2017. Some errors related to the (deprecated) COM api that's being used. But I will definitely test once I have that working again.